crates/compiler/parse/src/ast.rs

```diff
@@ -958,9 +960,21 @@ impl<'a, 'b> RecursiveValueDefIter<'a, 'b> {
                     expr_stack.push(&a.value);
                     expr_stack.push(&b.value);
                 }
-                Expect(condition, cont)
-                | Dbg(condition, cont)
-                | LowLevelDbg(_, condition, cont) => {
+                Expect(condition, cont) => {
+                    expr_stack.reserve(2);
+                    expr_stack.push(&condition.value);
+                    expr_stack.push(&cont.value);
+                }
+                Dbg(Some(expr)) => {
+                    expr_stack.reserve(1);
+                    expr_stack.push(&expr.value);
```

Comment:
No need to `reserve` if only pushing one item.

Reply:
Good point. Refactoring parsing removed this code anyways.